src/firefly/js/charts/dataTypes/FireflyHeatmap.js

```diff
@@ -26,18 +28,17 @@ import {getTraceTSEntries as genericTSGetter} from './FireflyGenericData.js';
 export function getTraceTSEntries({traceTS, chartId, traceNum}) {
     const {mappings} = traceTS;
 
-    if (!mappings) return {}; 
+    if (!mappings) return {};
 
     const {fireflyData, fireflyLayout} = getChartData(chartId) || {};
     // server call parameters
     const xbins = get(fireflyData, `${traceNum}.nbins.x`);
     const ybins = get(fireflyData, `${traceNum}.nbins.y`);
-    let maxbins = 10000;
-    let xyratio = get(fireflyLayout, 'xyratio', 1.0);
-    if (xbins && ybins) {
```

Comment:
I think we were talking into account xyratio that might be specified in the options when calculating the default number of bins. This feature is now lost.